ui/apps/everest/src/pages/db-cluster-details/cluster-overview/cards/db-details/advanced-configuration.tsx

```diff
@@ -1,47 +0,0 @@
-// everest
```

Comment:
this file was moved to `src/pages/db-cluster-details/cluster-overview/cards/db-details/advanced-configuration` folder